### PR TITLE
Update Acura logo

### PR DESCRIPTION
**Pull request by jglamp**
Acura logo

**Issue:** helps #10282

**Popularity metric:** 18005 on SimilarWeb

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Updated from homepage logo as per #10282

### CODE REVIEW

**Review by uncenter — DISMISSED** [review]

**Comment by jglamp** [comment]
From the looks of it, there are at least two Acura logos on the homescreen. There is one that has the wordmark to the side and another that has the wordmark at the bottom depending on the width of the screen displayed. I used the one with the wordmark to the side for this updated SVG.

**Comment by uncenter** [comment]
> From the looks of it, there are at least two Acura logos on the homescreen. There is one that has the wordmark to the side and another that has the wordmark at the bottom depending on the width of the screen displayed. I used the one with the wordmark to the side for this updated SVG.

So did I...

File: icons/acura.svg
[the remark this answers is not in the document]
Comment (uncenter) [reply icons/acura.svg:1]: Hmm still seeing differences:
Screenshot 2024-05-23 at 10 01 06 (Inkscape)

**Comment by jglamp** [comment]
If I compare the two icons, the one with the wordmark on the side (black) and the one with the wordmark on the bottom (red), I get this composite.

acura_comp

I can't lie, I'm kinda confused with this.

**Comment by adamrusted** [comment]
We just need to agree on one version to use, and compare against.
I'd suggest we use the one with the `aria-label` set to 'acura logo'.

**Comment by jglamp** [comment]
Alright, I've tried again with the `aria-label` set to acura-logo. Let me know how this one looks! I checked on Inkscape and it appeared that they were a good match, but I'm not too sure if that's true or not.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @jglamp! 🎉

**Pull request merged**